test/other/codesize/test_codesize_cxx_ctors1.size

```diff
@@ -1 +1 @@
-129166
+129206
```

Comment:
I would not expect to see any code size changes with this change?   At least not for tests that don't use ES6

Reply:
I'll rebase and try again.